src/main/java/core/basesyntax/type/Birds.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.type;
+
+public abstract class Birds {
+    private String species;
+    private boolean canFly;
+
+    public Birds(String species, boolean canFly) {
+        this.species = species;
+        this.canFly = canFly;
+    }
+
+    public void eat() {
```

Comment:
can we move method eat() in the common interface?

Reply:
done